<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectPlayerWindows</class>
 <widget class="QDialog" name="SelectPlayerWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WarCpp - SÃ©lection des joueurs</string>
  </property>
  <widget class="QLabel" name="labeljoueur1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>83</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/fond/images/panneau2.png);</string>
   </property>
   <property name="text">
    <string>Joueur1 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labeljoueur2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>124</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/fond/images/panneau2.png);
border-top-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Joueur2 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCommencer">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/fond/images/nouvelle_partie.jpg);</string>
   </property>
   <property name="text">
    <string>Commencer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRetour">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/fond/images/nouvelle_partie.jpg);</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonJ1">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonJ2">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>122</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirst">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>89</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>129</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AlbatorBot</string>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/fond/images/selectNation (532x333).jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>labeljoueur1</zorder>
  <zorder>labeljoueur2</zorder>
  <zorder>pushButtonCommencer</zorder>
  <zorder>pushButtonRetour</zorder>
  <zorder>radioButtonJ1</zorder>
  <zorder>radioButtonJ2</zorder>
  <zorder>labelNom</zorder>
  <zorder>labelFirst</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonRetour</sender>
   <signal>clicked()</signal>
   <receiver>SelectPlayerWindows</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
